examples/UnityClient/README.md

```diff
@@ -0,0 +1,37 @@
+# gRPC .NET client with Unity
+
+This example folder implements a
+[.NET gRPC](https://github.com/grpc/grpc-dotnet) client with Unity.
+
+## Usage
+
+In a terminal, go to `gnonative/goserver` and run the gRPC server:
+
+```bash
+go run . tcp -addr 127.0.0.1:7042
+```
+
+In Unity Hab, import this folder.
+
+In Unity, select the `SampleScene` in `Assets/Scenes`. Play the project, click
+on the button and look at the result in the text object.
+
+## Structure
+
+`Assets/Scenes/SampleScene.unity` is the Unity scene we use for the demo. We
+implemented a button to start an action, and a text object to print the result.
+
+`Assets/GnoClient.cs` is the script where lives the gRPC client. The `Start()`
```

Comment:
Small grammar fix:
"is the script where lives the gRPC client"
should be 
"is the script where the gRPC client lives"